### PR TITLE
Add images and tests for JDBC Sink, ES Sink and Replicator

### PR DESCRIPTION
**Pull request by arrawatia**

### CODE REVIEW

File: debian/base/Dockerfile [hunk 1]
```diff
@@ -31,14 +31,18 @@ ENV PYTHON_PIP_VERSION="8.1.2"
...
+ENV CONFLUENT_MINOR_VERSION="1"
+# REMOVE SNAPSHOT
+ENV CONFLUENT_PATCH_VERSION="0~SNAPSHOT"
 ENV CONFLUENT_VERSION="$CONFLUENT_MAJOR_VERSION.$CONFLUENT_MINOR_VERSION.$CONFLUENT_PATCH_VERSION"
 ENV CONFLUENT_DEB_VERSION="1"
-ENV CONFLUENT_DEB_REPO="http://packages.confluent.io"
+ENV CONFLUENT_DEB_REPO="https://s3-us-west-2.amazonaws.com/staging-confluent-packages-3.1.0-snapshot"
+
+# REMOVE SNAPSHOT
+ENV KAFKA_VERSION="0.10.1.0~SNAPSHOT"
```
Comment (theduderog) [inline debian/base/Dockerfile]: why define this in the base now?

**Comment by theduderog** [comment]
LGTM other than the tearDownClass stuff that seems to be commented out by mistake.  @samjhecht, are you good?  if so, i can fix the teardown() and merge.

File: tests/test_kafka_connect.py [hunk 2]
```diff
@@ -206,10 +220,10 @@ def setUpClass(cls):
         assert "PASS" in cls.cluster.run_command_on_service("kafka-host", KAFKA_READY.format(brokers=1))
         assert "PASS" in cls.cluster.run_command_on_service("schema-registry-host", SR_READY.format(host="localhost", port="8081"))
 
-    @classmethod
-    def tearDownClass(cls):
-        cls.machine.ssh("sudo rm -rf /tmp/kafka-connect-single-node-test")
-        cls.cluster.shutdown()
+    # @classmethod
```
Comment (arrawatia) [inline tests/test_kafka_connect.py]: Yes. I fixed it in the latest commit.

**Comment by arrawatia** [comment]
@theduderog I fixed that and also created a base connect image and renamed the replicator image to enterprise-replicator. Can you please review these changes ?

**Comment by theduderog** [comment]
your still awake!  LGTM

**Comment by arrawatia** [comment]
@theduderog Thanks.

**Pull request merged**